doc/monitoring/index.rst

```diff
@@ -64,6 +64,32 @@ in:
 
 It is good to outline that the main tool allowing users to observe and monitor a Ceph cluster is the **Ceph dashboard**. It provides graphics where the most important cluster and service metrics are represented. Most of the examples in this document are extracted from the dashboard graphics or extrapolated from the metrics exposed by the Ceph dashboard.
 
+Ceph daemon health metrics
+==========================
+
+The Ceph exporter provides a metric called ``ceph_daemon_socket_up`` that reports the health status of all Ceph daemons exposing an admin socket. These include the main Ceph daemons, encompassing any daemon capable of exposing performance counters.
+
+The ``ceph_daemon_socket_up`` metric indicates the health status of a Ceph daemon based on its ability to communicate through the admin socket, where a value of ``1`` means healthy, and ``0`` means unhealthy. Although a Ceph daemon might still be "alive" when it reports ``ceph_daemon_socket_up=0``, this situation highlights a significant issue in its functionality. As such, this metric serves as an excellent tool for detecting problems in any of the main Ceph daemons.
```

Comment:
Could `ceph_daemon_socket_up=1` (healthy) also result in unresponsive/unreachable daemons (e.g.: routing, network issues, firewalls, etc)? Meaning that we could have both false negatives and false positives?

Reply:
The metric only says that the daemon is not "answering" though the admin socket. 
That does not means that the daemon is "broken", is is not healthy but maybe it can continue doing his work. In any case is a situation that needs attention.